src/wanderer.ts

```diff
@@ -206,6 +219,40 @@ function guzzlrAbandonQuest() {
   }
 }
 
+function averageYrValue(location: Location) {
+  const monsters = Object.keys(getLocationMonsters(location))
+    .map((m) => toMonster(m))
+    .filter((m) => !m.attributes.includes("LUCKY") && !m.attributes.includes("ULTRARARE"));
+
+  if (monsters.length === 0) {
+    return 0;
+  } else {
+    return sumNumbers(
+      monsters.map((m) =>
+        sumNumbers(
+          itemDropsArray(m)
+            .filter((drop) => drop.type !== "c")
+            .map((drop) => garboValue(drop.drop))
+        )
+      )
+    );
+  }
+}
+
+function bestYrLocation(): { location: Location; value: number } {
+  const validLocations = $locations``
```

Comment:
I personally have a strong preference for `MafiaClass.all()` instead of using the empty template string thing. But I am a weirdo.

Reply:
This is not weird, you are right

Reply:
We should disallow this in libram, it's horrible!